jscomp/test/update_record_test.ml

```diff
@@ -29,6 +29,19 @@ let () =
   let  v = {a0=0;a1=0;a2=0;a3=0;a4=0;a5=0} in
   eq __LOC__ (v.a0 + 1) (f v).a0
 
+type t0 = {
+  invalid_js_id' : int;
```

Comment:
Do you already have a function to mangle these things?
There are bigger questions of how to add attributes to records a bit like we do with objects today, but that can wait. Just want to compile to valid code for now.

Reply:
Not mangling yet or reassignment, this is a place holder to remind us that we should handle it correctly at least in the ocaml world

Reply:
Looks like the generated code is OK?
```
function fff(x) {
  return /* record */{
          "invalid_js_id'": x["invalid_js_id'"] + 2 | 0,
          x: x.x
        };
}
```

Reply:
yes